.gitignore

```
@@ -4,6 +4,7 @@ dist
 # dependencies
 node_modules/
 .snowpack/
+package-lock.lock
```

```suggestion
package-lock.json
```
this should be `package-lock.json`